package.json.test.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-env jest */
 
 const packageJson = require('./package.json')
-const packageLockJson = require('./package-lock.json')
+// const packageLockJson = require('./package-lock.json')
 
 describe('because rollup 0.60 drops support for Internet Explorer 8', () => {
   describe('gulp-better-rollup', () => {
```

Comment:
we've dropped IE8 support, so we could delete these IE8 tests rather than commenting them out

Reply:
@oluoluoxenfree Can you delete the rollup test, and check everything still builds?

Reply:
@alexwybraniec would it not be better to delete this entire file? It's all about ie8 support.

Reply:
There are more ie8 mentions in the codebase which should probably be tidied up at some point.